### PR TITLE
Do kprove on circleci

### PR DESCRIPTION
**Pull request by yudetamago**
#42 

These branches work properly
* https://github.com/LayerXcom/verified-vyper-contracts/tree/erc20/dummy
* https://github.com/LayerXcom/verified-vyper-contracts/tree/erc721/dummy
* https://github.com/LayerXcom/verified-vyper-contracts/tree/test (the branch only for debugging CI)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,21 +1,70 @@
...
+                  - /erc20\/.*/
+                  - master
+                  - test
+        - erc721kprove:
+            filters:
+              branches:
+                only:
+                  - /erc721\/.*/
+                  - master
+                  - test
```
Comment (nrryuya) [inline .circleci/config.yml]: We still needs `test` branch?

Comment (yudetamago) [reply .circleci/config.yml]: Maybe so, I think.
Without it, the only way to do all tests and specs on CI is by merging to master branch.

Comment (nrryuya) [reply .circleci/config.yml]: How about `check_kprove_for_all_specs` branch?

Comment (yudetamago) [reply .circleci/config.yml]: ok 👍

Comment (yudetamago) [reply .circleci/config.yml]: ah, the branch includes pytest

Comment (yudetamago) [reply .circleci/config.yml]: something like `all_tests_and_specs` or `kprove_and_test_for_all` is proper?

Comment (nrryuya) [reply .circleci/config.yml]: >ah, the branch includes pytest　
ah, right.

`kprove_and_test_for_all` looks good.

**Comment by yudetamago** [comment]
:merge:

**Pull request merged**